docs/known_issues.rst

```diff
@@ -42,38 +40,38 @@ E.g.::
     >>> np.dot(q,q)
     285.0
     >>> np.hstack((q,q))
```

Comment:
To get this to pass, add `# doctest: +FLOAT_CMP` at the end (and similarly below) - really, there is no uncertainty in the floats here, of course, but `FLOAT_CMP` also deals with extra spaces, and we test with multiple numpy versions, including an older one that still does the spaces the old way...

p.s. But thanks for changing them all! It is much nicer if the format of the output matches what a user actually will see.

Reply:
Here, the `# doctest: +FLOAT_CMP` should be on this line (42), not line 40. I also think that once you do that, there will be a failure just a bit down.

Reply:
p.s. Ideally test this locally, but my guess it you'll need the stanza for every case in which there is an array output.

Reply:
Very sorry about that. I guess I misremembered which line this comment was on when I went to fix the issue. I wont let that happen again.
I tested this file with `pytest --doctest-plus --doctest-rst`, and it passed. This is how we're supposed to test .rst files, right? 6 other .rst files seem to be failing, although they were failing before I made any changes and I haven't touched them either, so I'm not sure what's going on there.
Anyway, I'll add any other missing `# doctest: +FLOAT_CMP`s where arrays are returned and fix this right away!

Reply:
@nzuckman - I usually just run `python3 setup.py test`, which perhaps has slightly different settings (I should know, but I fear I don't...).

Anyway, no worries; hopefully things will work!